src/scripting/zscript/zcc_parser.cpp

```diff
@@ -415,6 +415,24 @@ static void DoParse(int lumpnum)
 	for (unsigned i = 0; i < Includes.Size(); i++)
 	{
 		lumpnum = Wads.CheckNumForFullName(Includes[i], true);
+		if (lumpnum == -1 && Includes[i].IndexOf("./") == 0)//relative path resolving
+		{
+			FString fullPath = IncludeLocs[i].FileName.GetChars();//get full path, format 'wad:filepath/filename'
+			
+			size_t start = fullPath.IndexOf(":");//find first ':'
+
+			size_t end = fullPath.LastIndexOf("/");//find last '/'
+
+			if(start!=-1&&end!=-1)
+			{
+				FString path = fullPath.Mid(start + 1, end - start);//extract filepath from string
+				path += Includes[i].Mid(2);//add relative path
+
+				lumpnum = Wads.CheckNumForFullName(path, true);//check for relative include
+				if (lumpnum == -1) IncludeLocs[i].Message(MSG_ERROR, "Include script lump %s not found", path.GetChars());
```

Comment:
This line is almost useless, and the same error will be printed twice with relative and full paths.